### ISSUE TITLE
Hook up "Like" buttons

### ISSUE THREAD

**Issue by conrad-vanl** [NewSpring/Apollos#151]
## Acceptance Criteria

- [ ] Tapping on a "Like" buttons likes the content, state is persisted

## Issue todo:

- [ ] @Lepozepo what dependencies does this issue have that we still need to build? Mutations, queries, etc?

**Comment by Lepozepo** [comment]
@conrad-vanl if I remember correctly this information is coming from `Meteor.subscribe` in Holtzman. We need to:
- [ ] Build resolvers in Heighliner
- [ ] Build schema in Heighliner

**Comment by Lepozepo** [comment]
There may also be some db work behind this (a migration) since the new accounts system will not be reading into mongo and the likes are associated to MongoDB user _id's

**Comment by conrad-vanl** [comment]
@Lepozepo when you get a chance could you create the necessary issue(s) on the data side to unblock this?

**Comment by Lepozepo** [comment]
@conrad-vanl I think I can cover it in #137, I'll make another issue for handling the migration script
